locales/en-US/app.ftl

```diff
@@ -94,6 +94,7 @@ experimentListHeader = Pick your experiments!
 
 // An individual experiment in the listing of all Test Pilot experiments.
 [[experimentCard]]
+experimentCardFeedback = Feedback
```

Comment:
Can you add some context (and a comment) on what this string is? Is it a button label (e.g. "Give feedback?"), a link, something else?

For the latter, you might want to talk with Matjaz ;-)